py/desiutil/dust.py

```diff
@@ -0,0 +1,384 @@
+# Copied on Nov/20/2016 from https://github.com/kbarbary/sfdmap/ commit: bacdbbd
```

Comment:
Desiutil already has a license, and we need to ensure that all code is compatible with that license.  Maybe we were sloppy about that when the code was in desitarget, but desiutil is typically held to a much higher standard.

Reply:
I remain uncertain as to what to do about reusing code licensed in this manner. Now that you've brought this to my attention I certainly wouldn't merge this until we know the implications. I'm hoping @sbailey has some bright ideas.